akka-persistence-query/src/test/scala/akka/persistence/query/journal/leveldb/EventsByPersistenceIdSpec.scala

```diff
@@ -92,7 +91,7 @@ class EventsByPersistenceIdSpec extends AkkaSpec(EventsByPersistenceIdSpec.confi
       expectMsg(s"${3L}-deleted")
 
       val src = queries.currentEventsByPersistenceId("g1", 0L, Long.MaxValue)
-      src.map(_.event).runWith(TestSink.probe[Any]).request(1).expectComplete()
+      src.map(_.event).runWith(TestSink.probe[Any]).request(1).cancel()
```

Comment:
Does this still test anything now?

Reply:
I will have a look at it

Reply:
It was an error. I've reverted it and pushed an update.